src/commands/perform-tasks.ts

```diff
@@ -41,6 +41,7 @@ export class PerformTasksCommand extends BaseCliCommand<IPerformTasksCommandArgs
         command.option('--parameters [parameters]', 'parameters used when creating build tasks from tasks file');
         command.option('--organization-state-object [organization-state-object]', 'key for object used to load read-only organization state');
         command.option('--organization-state-bucket-name [organization-state-bucket-name]', 'name of the bucket that contains the read-only organization state');
+        command.option('--debug-templating [debug-templating]', 'when set to true the output of text templating processes will be stored on fisk', false);
```

Comment:
hmm, any reason why we need a separate flag for this?  why not just enable on the existing  `--verbose` flag?  Also doesn't the `print-tasks` command already process templates and output to local files?  

Other question..
* fisk/disk typo?
* there's already an existing `--verbose` flag so why not name it   `--verbose-templating` to be more consistent with existing command names?